bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -171,29 +193,39 @@ final class BagsManager(executorService: ExecutorService) {
                       if (!state.keys(key)) {
                         LOG.info(s"$bagsIdMsg successful. Results=${opt ? "Yes" | "No"}.")
                         applySwingResults(opt)
+                        clearStatus()
                       }
 
+                    // Note that we have to sleep after reporting a bags failure, as otherwise, there is no time for
+                    // the failure status message to be read before we change back to pending.
+
                     // We don't want to print the stack trace if the host is simply unreachable.
                     // This is reported only as a GenericError in a CatalogException, unfortunately.
                     case Failure(CatalogException((e: GenericError) :: _)) =>
                       LOG.warning(s"$bagsIdMsg failed: ${e.msg}")
+                      status(BagsStatus.Failed("Catalog lookup failed."))
+                      Thread.sleep(3000)
```

Comment:
It doesn't block the UI, as it isn't happening on the Swing thread.
I'm not sure a better way to allow `BagsStatus.Failed` error messages to persist for a period of readability without a sleep call somewhere: otherwise, they will immediately be flipped to `BagsStatus.Pending` in anticipation of the next lookup.